migration-engine/connectors/sql-migration-connector/Cargo.toml

```diff
@@ -1,33 +1,35 @@
 [package]
-name = "sql-migration-connector"
-version = "0.1.0"
 authors = ["Marcus Böhm <boehm@prisma.io>"]
 edition = "2018"
+name = "sql-migration-connector"
+version = "0.1.0"
 
 [dependencies]
-datamodel = { path = "../../../libs/datamodel/core" }
-datamodel-connector = { path = "../../../libs/datamodel/connectors/datamodel-connector" }
-migration-connector = { path = "../migration-connector" }
-native-types = { path = "../../../libs/native-types" }
-prisma-value = { path = "../../../libs/prisma-value" }
-sql-schema-describer = { path = "../../../libs/sql-schema-describer" }
-user-facing-errors = { path = "../../../libs/user-facing-errors", features = ["sql"] }
+datamodel = {path = "../../../libs/datamodel/core"}
+datamodel-connector = {path = "../../../libs/datamodel/connectors/datamodel-connector"}
+migration-connector = {path = "../migration-connector"}
+native-types = {path = "../../../libs/native-types"}
+prisma-value = {path = "../../../libs/prisma-value"}
+sql-schema-describer = {path = "../../../libs/sql-schema-describer"}
+user-facing-errors = {path = "../../../libs/user-facing-errors", features = ["sql"]}
 
 anyhow = "1.0.26"
 async-trait = "0.1.17"
 barrel = {git = "https://github.com/prisma/barrel.git", features = ["sqlite3", "mysql", "pg", "mssql"], branch = "mssql-support"}
-chrono = { version = "0.4" }
+chrono = {version = "0.4"}
 enumflags2 = "0.6.0"
+hex = "0.4"
```

Comment:
there is already a hex formatting function in the engine that doesn't introduce an external dependency, let me look for it

Reply:
That function was removed, but here it is reimplemented, let's maybe see if the tests pass with it: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b125c0d6b6253d270762200d1301b9ea

Otherwise I can take care of it in a subsequent PR.